crates/swc_ecma_minifier/tests/fixture/issues/7331/1/output.js

```diff
@@ -0,0 +1,5 @@
+export default function() {
+    var arg;
+    var arg;
```

Comment:
What's up with the double `var arg`? Is it because of the param and the function body's var?

Reply:
Yes. It can be dropped I guess, but it should be another PR IMO.